### PR TITLE
Update versions for v3.0.0-alpha1

### PR DESCRIPTION
**Pull request by caseydavenport**
## Todos

- [ ] Tests
- [ ] Documentation
- [ ] Release note

## Release Note

```release-note
None required
```

### CODE REVIEW

**Comment by caseydavenport** [comment]
I've opted not to duplicate the release notes from the RC, and just moved them into the actual alpha.

I think that's probably fine and good because nobody should be looking at the RC anyway now that alpha is out, and they are the same notes.

File: _data/versions.yml [hunk 1]
```diff
@@ -1394,6 +1394,42 @@ v3.0:
 
   components:
     felix:
+      version: 3.0.0-alpha1
+      url: https://github.com/projectcalico/felix/releases/tag/3.0.0-alpha1
+    typha:
+      version: v0.6.0-alpha1
+      url: https://github.com/projectcalico/typha/releases/tag/v0.6.0-alpha1
+    calicoctl:
+      version: v2.0.0-alpha1-rc2
```
Comment (bcreane) [inline _data/versions.yml]: this should be `v2.0.0-alpha1`

File: _data/versions.yml [hunk 2]
```diff
@@ -1394,6 +1394,42 @@ v3.0:
...
+      url: https://github.com/projectcalico/calicoctl/releases/tag/v2.0.0-alpha1-rc2
```
Comment (bcreane) [inline _data/versions.yml]: this should be `https://github.com/projectcalico/calicoctl/releases/tag/v2.0.0-alpha1`

File: _data/versions.yml [hunk 3]
```diff
@@ -1394,6 +1394,42 @@ v3.0:
...
+      download_url: https://github.com/projectcalico/calicoctl/releases/download/v2.0.0-alpha1-rc2/calicoctl
```
Comment (bcreane) [inline _data/versions.yml]: and this should be `https://github.com/projectcalico/calicoctl/releases/download/v2.0.0-alpha1/calicoctl`

**Pull request merged**